docs/artifacts/who-answers-to-that-name.html

```
@@ -0,0 +1,214 @@
+
+$ head -4 …/.venv/bin/aggregated-research
+  from kb_setup.research.cli import main             ← this repo, not the plugin
+
+$ echo $CLAUDE_PLUGIN_DATA
+  ~/.claude/plugins/data/codex-openai-codex          ← another plugin's dir
+
+$ …/cache/ray-manaloto/aggregated-research/eb3ba054916f/bin/aggregated-research --help
+  aggregated-research: nothing installed under …/codex-openai-codex
+
+CONTROL ARM — same binary, corrected env:
+$ CLAUDE_PLUGIN_DATA=…/data/aggregated-research-ray-manaloto  <same path> --help
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </same> ], start tag match failed [ <same path> ] on line 180.

(tag-pair)